doc/concepts/concepts.rst

```diff
@@ -1,13 +1,228 @@
+========
 Concepts
 ========
 
-These are the **why-docs** that discuss the design of MoveIt and this website.
+The following list major MoveIt concepts and components and describes how they tie in with each other. For tutorials, see the `tutorials <https://moveit.picknik.ai/>`_ page.
+
+.. image:: /_static/images/moveit_pipeline.png
+
+***********************
+The ``move_group`` node
+***********************
+
+The figure above shows the high-level system architecture for the primary node provided by MoveIt called [move_group](http://docs.ros.org/noetic/api/moveit_ros_move_group/html/move__group_8cpp.html). This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use.
+
+.. image:: /_static/images/move_group.png
+
+User Interface
+--------------
+
+The users can access the actions and services provided by ``move_group`` in three ways:
+
+- **In C++** - using the `move_group_interface <http://docs.ros.org/noetic/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroupInterface.html>`_ package that provides an easy to setup C++ interface to move_group
```

Comment:
This link should be updated to point to rolling (or, even better, `${moveit_release}`, if Sphinx has some way to expand variables).

Reply:
I'll update this to point to the `rolling` API docs once they are online. There's nothing better than `noetic` available for now.

Reply:
There is a way.  You use the variable `{DISTRO}`.  But as @jliukkonen said they don't exist yet so this is fine for now.